README.md

```diff
@@ -44,6 +44,13 @@ If you want to use LDAP, configure config/ldap.yml accordingly, then change the
     bin/rails console production
     u = User.new({ email: 'your@email.address', password: 'somepassword', password_confirmation: 'somepassword' }); u.agent = true; u.save!
 
+IMPORTANT Captcha Notice:
```

Comment:
Could you change this to a head? (Using `-----`) and rename it to "Configuring Captcha's" or something like that.